### PR TITLE
Set PR #39638 for Airflow 2.10

### PR DESCRIPTION
**Pull request by pankajkoti**
Set PR #39638 to be marked for 2.10 as we would like to consider
it as an improvement as suggested by @ephraimbuddy 

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/utils/db.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ class MappedClassProtocol(Protocol):
     "2.8.0": "10b52ebd31f7",
     "2.8.1": "88344c1d9134",
     "2.9.0": "1949afb29106",
-    "2.9.2": "0fd0c178cbe8",
+    "2.9.2": "686269002441",
```
Comment (ephraimbuddy) [inline airflow/utils/db.py:120]: this should only change on the version right?

Comment (hussein-awala) [reply airflow/utils/db.py:120]: This now points to the downstram of this revision (`down_revision = "686269002441"`) which is the last revision in 2.9.2

**Review by hussein-awala — APPROVED** [review]
LGTM

File: airflow/utils/db.py (hunk 1 above)
Comment (pankajkoti) [reply airflow/utils/db.py:120]: yes, true what Hussein explained.

**Review by Lee-W — APPROVED** [review]

**Review by ephraimbuddy — APPROVED** [review]

**Pull request merged**